web/src/components/pages/partner/partner.css

```diff
@@ -0,0 +1,168 @@
+.partner-main-container {
+    --type-color: var(--blue);
+}
+
+.partnerships-container {
+    display: flex;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: var(--desktop-width);
+    min-height: 100%;
+    transition: opacity var(--transition-duration-slow) var(--easing);
+    padding-top: 25px;
+
+    @media (--xs-down) {
+        flex-direction: column;
+    }
+
+    & .partnerships-header-text {
+        margin: 0px 20px 40px 20px;
+    }
+
+    & .page-heading {
+        font-size: 48px;
+
+        @media (--md-only) {
+            margin-top: 0;
+        }
+    }
+
+    & .partnerships-header-image {
+        @media (--xs-down) {
+            margin: 0 auto;
+        }
+
+        @media (--md-up) {
+            flex: 1;
+            text-align: right;
+        }
+
+        & img {
+            @media (--md-down) {
+                max-height: 220px;
+            }
+        }
+    }
+
+    p {
+        font-weight: 300;
+
+        @media (--xs-down) {
+            font-size: 18px;
+            width: 355px;
```

Comment:
Not sure we want to set exact widths considering the size of screen is variable and can be relative to the parent element